src/tests/curve.rs

```diff
@@ -355,6 +356,11 @@ pub fn hash_to_curve_test<G: CurveExt>() {
     }
 }
 
+pub fn endo_consistency_test<G: CurveExt>() {
```

Comment:
Why pub??

Reply:
Original `curve_testing_suite` was calling the function `endo_consistency_test`.
So, I had to pub the function to call itself inside the macro.
Now, the test logic is directly inside the macro.
https://github.com/privacy-scaling-explorations/halo2curves/pull/129/files#diff-e654b43be305459ee924b3b7b2cf6142dab1f359ceec19903abe85f591b10df8R426-R438
Hence, we dont have `pub fn endo_consistency_test` any more.